azerothcore/azerothcore-wotlk
# fix(DB/Creature): Great Goretusk spawned inside a tree in Redridge Mountains

## Changes Proposed:
-  Moved the spawn point to local coords 14.6 49.8, which is listed as a spawn point on https://classic.wowhead.com/npc=547/great-goretusk#map

## Issues Addressed:

- Closes #6116
- Closes chromiecraft/chromiecraft#729

## SOURCE:

https://classic.wowhead.com/npc=547/great-goretusk#map

## Tests Performed:

- See below


## How to Test the Changes:


1. Run ```SELECT * FROM `acore_world`.`creature` WHERE `guid` IN (6707);``` The result should have:
position_x = -9296.069336, position_y = -1887.173462, position_z = 78.376251
2. Use ```go creature 6707``` to teleport to the changed creature. It should no longer be in the tree.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
